handlers/auth.go

```diff
@@ -24,3 +24,19 @@ func Auth(secret string) gin.HandlerFunc {
 		c.Set("user", user)
 	}
 }
+
+func getUserFromJwt(webtoken, secret string) (*models.User, error) {
+	token, err := jwt.Parse(webtoken, func(token *jwt.Token)(interface{}, error){
```

Comment:
What is `webtoken`? It looks like this function is mostly the same as the anonymous one returned above; trying to think if there's a way that we can combine them. (I like that this has been abstracted and wonder if it should also be used above, is what I'm saying.)